components/host-dashboard/AddFundsModal.js

```diff
@@ -121,6 +121,10 @@ const AddFundsModal = ({ host, collective, ...props }) => {
           const hostFee = Math.round(values.amount * (hostFeePercent / 100));
           const platformFee = Math.round(values.amount * (platformFeePercent / 100));
 
+          // From the Collective page we pass host and collective as API v1 objects
+          // From the Host dashboard we pass host and collective as API v2 objects
+          const canAddHostFee = host.plan.hostFees && collective.id !== host.id;
```

Comment:
Maybe we should move that up above `const defaultHostFeePercent = collective.hostFeePercent;` and update that line to:

```es6
const defaultHostFeePercent = canAddHostFee ? collective.hostFeePercent : 0;
```

That way even if the collective somehow  has a `hostFeePercent` set it will be ignored.

Reply:
I integrated the feedback!